### PR TITLE
Enable trading as name to be piped into questions

### PR DESCRIPTION
**Pull request by benjefferies**
### What is the context of this PR?
Create a way of dynamically populated trad_as or ru_name from the metadata. See [trello](https://trello.com/c/JLIbPCgI/975-enable-trading-as-name-to-be-piped-into-questions)

### How to review 
- Open qbs with trad_as and ru_name
Expect to see trad_as populated

- Open qbs with only ru_name
Expect to see ru_name populated

- Open mwss with trad_as and ru_name
Expect to see trad_as populated

- Open mwss with only ru_name
Expect to see ru_name populated

### CODE REVIEW

File: app/templating/schema_context.py [hunk 1]
```diff
@@ -54,4 +54,5 @@ def _build_respondent(metadata):
     return {
         "ru_name": metadata["ru_name"],
         "trad_as": metadata["trad_as"],
+        "display_name": metadata["trad_as"] or metadata["ru_name"],
```
Comment (LJBabbage) [inline app/templating/schema_context.py]: I reckon we use trad_as_or_ru_name, just makes it clear

**Review by LJBabbage — CHANGES_REQUESTED** [review]

File: app/templating/schema_context.py (hunk 1 above)
Comment (benjefferies) [reply app/templating/schema_context.py]: Done

**Review by LJBabbage — APPROVED** [review]

File: gulpfile.babel.js [hunk 2]
```diff
@@ -181,6 +181,10 @@ gulp.task('lint:json', () => {
       quiet: false,
       fail: true
     }))
+    .on('error', (err) => {
```
Comment (jonnyshaw89) [inline gulpfile.babel.js:184]: Shouldn't this be a separate PR?
I don't really care but its a comment you often raise :)

Comment (benjefferies) [reply gulpfile.babel.js:184]: Yes but didn't want to create a ticket for it 😆. Do you want me to?

Comment (jonnyshaw89) [reply gulpfile.babel.js:184]: No. I was just being annoying.

**Review by jonnyshaw89 — APPROVED** [review]

**Pull request merged**